### PR TITLE
Update BigDecimal in gremlin-python statics to be consistent with server definition

### PR DESCRIPTION
**Pull request by xiazcy**
Added a value property to BigDecimal so it can be properly serialized in Gremlin Lang to the server for GraphBinary. 

Added a `to_bigdecimal` function to help creating BigDecimal in python and for GraphSON. 

Currently targeting the test branch, will rebased once that PR is merged.

### CODE REVIEW

**Comment by Cole-Greer** [comment]
VOTE +1

**Comment by xiazcy** [comment]
VOTE +1

File: gremlin-python/src/main/python/gremlin_python/statics.py [hunk 1]
```diff
@@ -74,12 +75,35 @@ def __init__(self, gremlin_type):
...
 
+    @property
+    def value(self):
+        self._as_decimal = Decimal(self.unscaled_value)
+        precision = len(self._as_decimal.as_tuple().digits)
+        decimal.getcontext().prec = precision
+        return self._as_decimal.scaleb(-self.scale)
+
+"""
+Create a BigDecimal from a number that can be converted to a Decimal. Note precision may be lost in the convertion.
```
Comment (kenhuuu) [inline gremlin-python/src/main/python/gremlin_python/statics.py:94]:
```suggestion
Create a BigDecimal from a number that can be converted to a Decimal. Note precision may be lost during the conversion.
```
Nit

File: gremlin-python/src/main/python/gremlin_python/statics.py [hunk 2]
```diff
@@ -74,12 +75,35 @@ def __init__(self, gremlin_type):
...
+    """
     def __init__(self, scale, unscaled_value):
         self.scale = scale
         self.unscaled_value = unscaled_value
 
+    @property
+    def value(self):
+        self._as_decimal = Decimal(self.unscaled_value)
+        precision = len(self._as_decimal.as_tuple().digits)
+        decimal.getcontext().prec = precision
```
Comment (kenhuuu) [inline gremlin-python/src/main/python/gremlin_python/statics.py:90]: Does this cause the precision on a global level?

Comment (xiazcy) [reply gremlin-python/src/main/python/gremlin_python/statics.py:90]: Actually it does, I'll update to either reset or use local context.

Comment (xiazcy) [reply gremlin-python/src/main/python/gremlin_python/statics.py]: Updated with using a local context.

**Comment by kenhuuu** [comment]
VOTE +1

**Pull request merged**